modules/ROOT/pages/docker-ldap-synchronizer.adoc

```diff
@@ -0,0 +1,96 @@
+= Deploy LDAP Synchronizer with Docker
```

Comment:
This page is missing from the taxonomy.
Also, IMO, it would make more sense to have everything on the same page `ldap-synchronizer.adoc`

Reply:
ok